test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1447,3 +1832,36 @@ func getTunedStructuredData(profile *performancev2.PerformanceProfile) *ini.File
 	Expect(err).ToNot(HaveOccurred())
 	return cfg
 }
+
+func deleteTestPod(testpod *corev1.Pod) {
+	// it possible that the pod already was deleted as part of the test, in this case we want to skip teardown
+	err := testclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(testpod), testpod)
+	if errors.IsNotFound(err) {
+		return
+	}
+
+	err = testclient.Client.Delete(context.TODO(), testpod)
+	Expect(err).ToNot(HaveOccurred())
+
+	err = pods.WaitForDeletion(testpod, pods.DefaultDeletionTimeout*time.Second)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+//checkCpuGovernors Checks power settings of the cpus
+func checkCpuGovernors(cpus []int, targetNode *corev1.Node, pm_qos string, governor string) error {
```

Comment:
nit: a better name might be checkCpuGovernorsAndResumeLatency

Reply:
@bartwensley addressed this in the latest patch